broker/providers/ansible_tower.py

```diff
@@ -21,6 +22,8 @@
 from broker import helpers
 from broker.providers import Provider
 
+YAML().register_class(awxkit.utils.PseudoNamespace)
```

Comment:
Actually, it's probably better to act on `helpers.yaml` here
```suggestion
helpers.yaml.register_class(awxkit.utils.PseudoNamespace)
```